config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -75,11 +91,117 @@ processors:
         action: insert
       - key: aws.ecs.task.id
         action: delete
-      - key: TaskDefinitionFamily
+      - key: Family
```

Comment:
Same here- we should not change this dimension.

Reply:
Updated